README.md

```diff
@@ -1 +1,131 @@
-# portfolio-project
\ No newline at end of file
+<a name="readme-top"></a>
+# portfolio-project
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```

Comment:
Kindly, I suggest you remove the live demo section if your project does not have deployed yet 😄